### DISCUSSION TITLE
Implement stacking for notifications

### DISCUSSION THREAD

**Discussion by anaximeno** [linuxmint/discussions#217] [Ideas]
This would come with an option to insert the number of notifications to stack (somthing like min. 1 to max. 5)

**Comment by rcalixte** [comment]
I've actually looked into this in tandem with a setting to display notifications on more than one display simultaneously, effectively, a different form of showing multiple notifications. Both have proved more difficult to implement for now but I do like the idea. (I have a separate pull request open for other tweaks to notifications as well.)

**Reply by anaximeno** [reply]
Yeah, it would be a nice feature, I've been thinking of implementing something like it for some time so I decided to put it here in case someone wants to implement it before I have time to take into it. I've also been following the pull requests you made related to notification, they would be great updates as well.

**Comment by anaximeno** [comment]
Probably it would have to be implemented per monitor if the notification is set to "active window", like each monitor to have it's stack, this way the behavior would be consistent if the second (or the third+), notification is sent after moving the cursor to a different monitor than the one it was when the first notification was sent.
